tests/app/storage/test_questionnaire_storage.py

```diff
@@ -4,37 +4,37 @@
 from sqlalchemy.exc import IntegrityError
 
 from app import settings
-from app.storage.questionnaire_storage import QuestionnaireStorage
+from app.storage.encrypted_questionnaire_storage import EncryptedQuestionnaireStorage
 
 
 class TestQuestionnaireStorage(unittest.TestCase):
 
     def setUp(self):
         settings.EQ_SERVER_SIDE_STORAGE_DATABASE_URL = "sqlite://"
-        self.storage = QuestionnaireStorage("1")
+        self.storage = EncryptedQuestionnaireStorage("user_id", "user_ik", "pepper")
 
     def test_store(self):
-        data = {'test': 'test'}
+        data = 'test'
         self.assertIsNone(self.storage.add_or_update(data))
-        self.assertIsNotNone(self.storage._get())  # pylint: disable=protected-access
+        self.assertIsNotNone(self.storage._find_questionnaire_state())  # pylint: disable=protected-access
```

Comment:
Not really, the test was changed as a result of a rename of the `_get` method. I'll change this so that it tests against the public interface.